src/components/TopbarViewer/TopbarViewer.tsx

```diff
@@ -104,7 +110,7 @@ export const ViewerTopbar: React.FC = () => {
             Start publishing
           </Button>
         )}
-        {!isFocused && !smMatch && (
+        {!isFocused && !mdMatch && (
           <IconButton>
```

Comment:
I'll look at the code later. Just noticed one thing - this button doesn't do anything :P Could you also add here `onClick={() => openInNewTab(absoluteRoutes.studio.index())}`?

Reply:
Yeah good catch, missed that. Those should actually be native links instead, will fix tomorrow